grasshopper.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.1.4",
+    "url": "https://grasshopper.coding.kiwi/download/1-1-4-grasshopper-1-1-4-exe#/grasshopper.exe",
+    "hash": "5a22c59b4ab0face21e6c6ee9ee2af2a915ae5a3a5866aa18b3b85065bfc91a8",
+    "bin": "grasshopper.exe",
+    "autoupdate": {
+        "url": "https://grasshopper.coding.kiwi/download/$dashVersion-grasshopper-$dashVersion-exe#/grasshopper.exe",
+        "hash": {
+            "mode": "json",
+            "jp": "$.files[1].sha512",
+            "url": "https://grasshopper.codingkiwi.de/client/latest.json"
+        }
+    },
+    "homepage": "https://grasshopper.coding.kiwi",
+    "checkver": {
+        "url": "https://grasshopper.codingkiwi.de/client/latest.json",
+        "jp": "$.version"
+    },
+    "license": "Freeware",
+    "description": "The free and easy SSH client"
```

Comment:
```suggestion
    "version": "1.1.4",
    "description": "SSH client",
    "homepage": "https://grasshopper.coding.kiwi",
    "license": {
        "identifier": "Freeware",
        "url": "https://grasshopper.coding.kiwi/faq#free"
    },
    "architecture": {
        "64bit": {
            "url": "https://grasshopper.coding.kiwi/download/1-1-4-grasshopper-Setup-1-1-4-exe#/dl.7z",
            "hash": "d8950908ea59a08823d9faf1ea7dcfa74bdfbfcc59bf33fbbfca3db12ba71aaf",
            "pre_install": [
                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
            ]
        }
    },
    "bin": "graSSHopper.exe",
    "shortcuts": [
        [
            "graSSHopper.exe",
            "graSSHopper"
        ]
    ],
    "checkver": {
        "url": "https://grasshopper.codingkiwi.de/client/latest.json",
        "jsonpath": "$.version"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://grasshopper.coding.kiwi/download/$dashVersion-grasshopper-Setup-$dashVersion-exe#/dl.7z"
            }
        }
    }
```

Reply:
yeah, see, stuff like that should actually be documented, not just left in an issue text and assume that i'm gonna find it in one of several thousand issues, lol.